projects/sandbox/train/train/validation.py

```diff
@@ -117,9 +118,13 @@ class Validator:
     tracker: LocalTracker
     background: torch.Tensor
     waveforms: torch.Tensor
+    asd_estimator: Callable
+    whitener: Callable
```

Comment:
Where exactly is the validation data whitened? It doesn't look like this is used anywhere.

Reply:
Yeah good catch this was screwing things up, should be fixed now